website/cue/reference/components/sources/datadog_agent.cue

```diff
@@ -232,14 +232,23 @@ components: sources: datadog_agent: {
 					metrics.url: http://"<VECTOR_HOST>:<SOURCE_PORT>" # Use https if SSL is enabled in Vector source configuration
 				```
 
+				In order to send traces the [Datadog Agent](\(urls.datadog_agent_doc)) configuration must be updated with the
+				following options:
+
+				```yaml
+				apm_config:
+					apm_dd_url: http://"<VECTOR_HOST>:<SOURCE_PORT>" # Use https if SSL is enabled in Vector source configuration
```

Comment:
Isn't there a `vector.traces.url` now?

Reply:
I couldn't find it but I'll take another look if you think it exists 😄

Reply:
I _think_ it became a thing in a fairly recent agent version. Let me take a look too.

Reply:
https://cs.github.com/DataDog/datadog-agent/blob/a728670757b568e7cb4e1deb7de6fad0b80cbdd2/releasenotes/notes/vector-traces-override-bebe167f286f7a3d.yaml?q=vector